### PR TITLE
Adding log directory param and symlink

### PR DESCRIPTION
**Pull request by Zhornovyi**
### What does it do?

Initialize log directory from console parameters or create it inside the project folder. Then creates a symbolic link at  /home/ga_zoho_logs

### Checklist
- [ ] Pipeline set to `Ready To Deploy` or `Review/QA`
- [ ] Pull request is prepared: Added to sprint, added yourself as assignee, Zenhub task is linked, reviewers are added
- [ ] PR is not very big (about 200 lines of change)
- [ ] Doesn't create any new `FIXME` or `TODO` comments. If not, please explain or create issues to resolve it.
- [ ] Doesn’t deliver half-baked changes
- [ ] Tested manually or with a new unit test
- [ ] Follows best practice rules

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -47,3 +40,21 @@
...
+        if ex.errno != errno.EEXIST:
+            logging.exception("Problems with creating log directory", exc_info=ex)
+            init_logdir('./logs')
+            return False
+        else:
+            print('Log directory already created at ' + logpath)
+    else:
+        print("Successfully created the logs directory at: \n" + logpath)
+    LOG_DIR = logpath
+    symlink(logpath, dest)
```
[the remark this answers is not in the document]
Comment (Zhornovyi) [reply config.py]: done at 5c9152e

File: README.md [hunk 2]
```diff
@@ -51,7 +51,12 @@ from Zoho CRM
 `-port`: port for Zoho CRM notifications Webhook server. By default it's 80.
 To use not 80 port you will need to do additional proxy server set up.
 
-`-log`: mode of logging handling. To enable logging to console pass 'console'. By default it's 'file'
+`-logmode`: mode of logging handling. To enable logging to console pass 'console'. By default it's 'file'
+
+`-logpath`: path of storing logs. By dafault it's script path + 'logs' directory.
```
Comment (andrii-chumak) [inline README.md:56]: Type error in word 'default'

Comment (andrii-chumak) [inline README.md:56]: Type error in word 'default'

**Review by andrii-chumak — COMMENTED** [review]
1 minor typo error
Pls, provide demo with logging

**Pull request closed without merging**